### DISCUSSION TITLE
Output via bluetooth HID profile?

### DISCUSSION THREAD

**Discussion by snst** [darthcloud/BlueRetro#94] [Ideas]
Would it be possible that BlueRetro also acts as a bluetooth HID device?
E.g. you connect a PS3 controller via BT sisaxis to BlueRetro and BlueRetro connects via BT HID profile to an android device?

**Comment by darthcloud** [comment]
Not impossible I guess but definitely not something I will work on.

Your input latency would be terrible going in the air twice.
